tools/heir-opt.cpp

```diff
@@ -205,6 +206,9 @@ void convertToDataObliviousPipelineBuilder(OpPassManager &manager) {
 void polynomialToLLVMPipelineBuilder(OpPassManager &manager) {
   // Poly
   manager.addPass(createElementwiseToAffine());
+  manager.addPass(
+      ::mlir::heir::polynomial::to_mod_arith::createPolynomialToModArith());
+  manager.addPass(mod_arith::createModArithToArith());
   manager.addPass(::mlir::heir::polynomial::createPolynomialToStandard());
```

Comment:
Do we still need `PolynomialToStandard` here?

Reply:
`PolynomialToModArith` only handles add/mul_scalar/from_tensor/to_tensor/constant, we still need `PolynomialToStandard` to handle mul/ntt/intt/monomial things